app/scripts/models/reliers/oauth.js

```diff
@@ -159,7 +159,13 @@ define([
           //jshint camelcase: false
           // server version always takes precedent over the search parameter
           self.set('redirectUri', serviceInfo.redirect_uri);
+          self.set('termsUri', serviceInfo.terms_uri);
+          self.set('privacyUri', serviceInfo.privacy_uri);
+          self.set('trusted', serviceInfo.trusted);
```

Comment:
I guess we could just shamelessly copy whatever requirement big-G put on their reliers